test/test_main.py

```diff
@@ -0,0 +1,81 @@
+import sys
+from collections import OrderedDict
+sys.path.append('..')
+sys.path.append('./translator/GCP_cloud_function/cloud_function')
+from unittest.mock import MagicMock, patch, call, Mock
+sys.modules['icone_translator'] =Mock()
+import main
+import icone_translator
+import os
+import json
+
+
+@patch('urllib.request')
+def test_get_ftp_file(request):
+    test_url='fake url'
+    test_output=main.get_ftp_file(test_url)
+    request.urlopen.assert_called_with(test_url)
+    assert True
```

Comment:
We're not gaining anything by asserting a boolean, let's remove.

Reply:
i removed it pushed the updaated code.

Reply:
Seems to still be here in the [dev branch](https://github.com/CDOT-CV/Work_Zone/blob/db9ae42bc5816f3322e8aec38f16acfd5dc15ed1/test/test_main.py#L18)

Reply:
sorry for that and i removed I just pushed the updated code to dev again.